### PR TITLE
Updated Sidebar to include Community Components

### PR DESCRIPTION
**Pull request by jingningzhang1**
This PR is to include the Dash Community Components page on the Dash Docs.

Post-merge checklist:

The master branch is auto-deployed to `dash.plotly.com`.
Once you have merged your PR, wait 5-10 minutes and check dash.plotly.com
to verify that your changes have been made.

- [x] I understand

### CODE REVIEW

File: dash_docs/chapter_index.py [hunk 1]
```diff
@@ -666,8 +666,24 @@ def component_list(
...
+            },
 
+            {
+                'name': 'Dash Community Components',
+                'chapters': [
+                    {
+                        'url': 'https://plotly.com/dash-community-components/',
+                        'name': 'Components List',
+                        'description': (
+                            'A listicle of selected components from the Dash Community'
```
Comment (chriddyp) [inline dash_docs/chapter_index.py:678]:
```suggestion
                            'A listicle of selected components from the Dash Community '
```

Comment (chriddyp) [reply dash_docs/chapter_index.py:678]: image

**Review by chriddyp — APPROVED** [review]
💃 once the space issue is fixed. Thanks for this!

**Review by xhlulu — APPROVED** [review]
Same thing as Chris; 💃  once typo is corrected :)

File: dash_docs/chapter_index.py (hunk 1 above)
Comment (rpkyle) [reply dash_docs/chapter_index.py:678]: @jingningzhang1 If I could make a suggestion, maybe reword as we currently have "Dash Community" appearing twice in this sentence. One possible alternative:

"A listicle of selected Dash components created by users in our community forum."

or

"A listicle of selected components created by users within the Dash Community."
